src/index.d.ts

```diff
@@ -982,6 +982,121 @@ declare namespace admin.remoteConfig {
    */
   type RemoteConfigParameterValue = ExplicitParameterValue | InAppDefaultValue;
 
+  /**
+   * Interface representing a Remote Config template version.
+   * Contains all metadata about a particular version of the Remote Config template.
+   * All fields are set at the time the specified Remote Config template was written.
+   */
+  export interface Version {
+    /**
+     * The version number of the version's corresponding Remote Config template. Output only.
+     */
+    versionNumber?: string | number;
+    
+    /**
+     * The Remote Config template creation timestamp in RFC3339 UTC "Zulu" format,
```

Comment:
We generally expose timestamps in UTC string format in this library. We may want to stick to that convention here also.